src/TaskLineRenderer.ts

```diff
@@ -177,6 +177,16 @@ function addTooltip({
             recurrenceDiv.setText(`${taskModule.recurrenceSymbol} ${task.recurrence.toText()}`);
         }
 
+        if (task.createdDate) {
+            const createdDateDiv = tooltip.createDiv();
+            createdDateDiv.setText(
+                toTooltipDate({
+                    signifier: taskModule.createdDateSymbol,
+                    date: task.createdDate,
+                }),
+            );
+        }
+
```

Comment:
Thank you for adding this.
Before it is released, we would want to have a test for it.

Reply:
I don't find any tests in suites for the tooltips. Can you give a hint?

Reply:
Sure, so some general background:

In this repo, usually the names of test files are based on the source file name.

So I am just searching for `TaskLineRenderer.test.ts`

And Yes, it does exist.

So it would be a case of looking at the patterns of the tests in that file, and seeing if you can see how to add a new test, or update an existing one, as appropriate, to make sure that the change you made is being executed in the test file, and the result is tested.